recipes-synthesis/litex/litex.bb

```diff
@@ -2,7 +2,7 @@ SUMMARY = "LiteX is a FPGA design/SoC builder that can be used to build cores, c
 HOMEPAGE = "https://github.com/enjoy-digital/litex"
 SECTION = "devel/hdl"
 LICENSE = "BSD-2-Clause"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=0ce55a94e98cbec2110586bb5264bd9a"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=e2abad236488e3dd7fc3c9f12deb2a7c"
```

Comment:
You appear to have an override in your meta-ptx-fpga layer that forces the litex recipe to version 2022.04 (https://github.com/pengutronix/meta-ptx-fpga/blob/master/recipes-synthesis/litex/litex.bbappend#L7), this version has a different `LICENSE` file, you will need to include this `LIC_FILES_CHKSUM` in your bbappend.

Since this is not required for the meta-hdl recipe, please remove this change from the pull request.

Reply:
Damn, you're right. I'll remove it.